resources/cdk/glue_role_bucket/setup.d.ts

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
```

Comment:
Are there supposed to be changed in the CDK for glue?

Reply:
No changes here.